cfg/cis-1.5/master.yaml

```diff
@@ -282,7 +282,7 @@ groups:
 
       - id: 1.1.11
         text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Scored)"
-        audit: ps -ef | grep $etcdbin | grep -- --data-dir | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
+        audit: /bin/ps -ef | /bin/grep $etcdbin | /bin/grep -- --data-dir | /bin/sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
```

Comment:
`xargs` has same problem but there isn't `/bin/xargs` in alpine(it's in `/usr/bin` but overwrite by mount from host). Maybe we should change the mount dir like still mount `/usr/bin` from host but use `/usr/local/sbin` inside container, use this way we can both use `kubectl` command from host and use all builtin commands from alpine.